File: CodeForces/Training 1500/W.cpp
#include<bits/stdc++.h>
using namespace std;
#define endl '\n'
using ll = long long;
using ld = long double;
#define pb push_back
#define sz size()

void solver(){
    int n; cin>>n;
    string a, b; cin>>a>>b;
    int ans = 0;

    char last = a[0];
    int count = 1;
    for(int i = 1; i < n; i++){
        if(a[i] == last) count++;
        else{
            ans += count - 1;
            count = 1;
            last = a[i];
        }
    }

    ans += count - 1;

    cout<<ans<<endl;
}

int main(){
    ios_base::sync_with_stdio(0);cin.tie(NULL);
    int t = 1;
    cin>>t;
    while(t--){
        solver();
    }

    return 0;
}

File: CodeForces/Training 1500/W.kt
import java.util.*

fun solver() {
    val n = readLine()!!.toInt()
    val a = readLine()!!
    val b = readLine()!!

    var ans = 0
    var last = a[0]
    var count = 1

    for (i in 1 until n) {
        if (a[i] == last) {
            count++
        } else {
            ans += count - 1
            count = 1
            last = a[i]
        }
    }

    ans += count - 1
    println(ans)
}

fun main() {
    val t = readLine()!!.toInt()
    repeat(t) {
        solver()
    }
}
